src/components/Table/index.tsx

```diff
@@ -142,15 +142,20 @@ export const Table = <TData, TValue>({
 
   const TableBodyElement = useMemo(() => {
     if (deferredLoading || !deferredRows?.length) {
+      const isSearchTable = window.location.pathname.includes('/elephant/search')
       return (
         <TableRow>
           <TableCell
             colSpan={columns.length}
             className='h-24 text-center'
           >
-            {deferredLoading
-              ? 'Laddar...'
-              : 'Inga planeringar funna.'}
+            {isSearchTable
+              ? deferredLoading
+                ? ''
+                : 'Inga poster funna.'
+              : deferredLoading
+                ? 'Laddar...'
+                : 'Inga planeringar funna.'}
```

Comment:
Since 'Inga planeringar funna' is the wrong phrase in most of occasions, we might as well change to 'Inga poster funna' for all? Its more general.

Reply:
fixed in new commit